metadata-ingestion/src/datahub/ingestion/glossary/datahub_classifier.py

```diff
@@ -66,6 +83,9 @@ class Config:
         description="Factors and their weights to consider when predicting info types",
         alias="prediction_factors_and_weights",
     )
+    ExcludeName: Optional[ExcludeNameFactorConfig] = Field(
```

Comment:
```suggestion
    ExcludeName: Optional[List[str]] = Field(default=None,  description="List of exact column names to exclude from classification for this info type")
```

Reply:
We do not need separate class as it can be represented as list of  str directly..